bot/exts/evergreen/youtube.py

```diff
@@ -0,0 +1,151 @@
+import logging
+from dataclasses import dataclass
+from html import unescape
+from typing import List, Optional
+from urllib.parse import quote_plus
+
+from discord import Embed
+from discord.ext import commands
+from discord.utils import escape_markdown
+
+from bot.constants import Colours, Emojis, Tokens
+
+log = logging.getLogger(__name__)
+
+KEY = Tokens.youtube
```

Comment:
Please remove this wrapper

Reply:
^ +1

Reply:
@ks129 specifically you meant the token wrapper, correct?